sectionctl.go

```diff
@@ -48,6 +48,7 @@ func bootstrap(c CLI, ctx *kong.Context) {
 	log.SetOutput(filter)
 
 	switch {
+	case ctx.Command() == "version":
```

Comment:
Worth being more explicit about the empty case?
```suggestion
	case ctx.Command() == "version":
		// bypass auth check for version command
```

Reply:
That's a great call, I'll add that now.